docs/jwst/error_propagation/main.rst

```diff
@@ -0,0 +1,374 @@
+For more details on this pipeline see `Calwebb_coron3
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/pipeline/calwebb_coron3.html>`_
+
+
+Outlier_detection 
+~~~~~~~~~~~~~~~~~
+This step uses the ERR array to make a local noise model, based on the readnoise
+and calibration errors of earlier steps in the pipeline. This step does not modify
+the ERR array or any of the VAR arrays. 
+
+
+Stage 3 Time-Series Observation (TSO) Processing Pipeline - calwebb_tso3
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+The stage 3 TSO pipeline is to be applied to associations of calibrated TSO
+exposures.  The steps in this pipeline are outlier_detection, tso_photometry,
+extract_1d, and white_light.  The only one of these steps that either uses or
+modifies variance/error arrays that are propagated through the pipeline is
+outlier_detection, described here:
+
+Outlier_detection 
+~~~~~~~~~~~~~~~~~
+This step uses the ERR array to make a local noise model, based on the readnoise
+and calibration errors of earlier steps in the pipeline. This step does not modify
+the ERR array or any of the VAR arrays.
+
+Stage 3 WFS&C Processing pipeline - calwebb_wfs-image3
+++++++++++++++++++++++++++++++++++++++++++++++++++++++
+Stage 3 processing of Wavefront Sensing and Control (WFS&C) images is only
+performed for dithered pairs of WFS&C exposures. The processing applied is not 
+truly a “pipeline”, but consists only of the single wfs_combine step. This step
+could modify variance/error arrays, but only if the optional 'do_refine' is set
+to True (which is *NOT* the default in pipeline use). In this case the "refined 
+algorithm" will be used, and the ERR array values will be altered on output.
+
+
+Other pipelines
+---------------
+In addtion to the Level 1, 2, and 3 pipelines discussed so far, there are 2 other 
+pipelines- Dark Processing, and Guide Star Processing.
+
+Dark Processing pipeline - calwebb_dark
++++++++++++++++++++++++++++++++++++++++
+The Dark Pipeline applies basic detector-level corrections to all dark exposures.
+For details on thie pipeline see `Calwebb_dark
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/pipeline/calwebb_dark.html>`_
+All of the steps in this pipeline are group_scale, dq_init, saturation, ipc, superbias, 
+refpix, linearity, and rcsd.  The only one of these steps that either uses or
+modifies variance/error arrays that are propagated through the pipeline is
+group_scale, described here:
+
+Group_scale
+~~~~~~~~~~~
+This step is applied after the ramp_fitting step, and applies to both the rate and
+rateints products from ramp_fit. The correction is applied to the ERR, and will
+also be applied to the VAR_POISSON, and VAR_RNOISE arrays.  The details of the 
+calculations can be found at `Group Scale
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/group_scale/description.html>`_
+
+
+Guide Star Processing pipeline - calwebb_guider
```

Comment:
This is the same info already included for the ``flat_field`` step in the Stage 2 section above and hence isn't needed.